### ISSUE TITLE
Let's check how our components hold agains w3 wai-aria patterns

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#3110]
https://www.w3.org/WAI/ARIA/apg/patterns/

Let's check our components one by one and see how it compares.

Maybe you can stop after a couple of checks if it's too obvious we can't support that.

Result on this task should be a list of requirements for every component + some reasoning.

**Comment by kushich** [comment]
moved to https://github.com/epicmaxco/vuestic-ui/issues/3223
